docs/docs/spec.md

```diff
@@ -23,10 +23,10 @@ In this doc, we aim not to refer to whether a type "is nullable." Instead, we
 draw some distinctions, creating at least four kinds of "Is it nullable?"
 questions we can ask for any given type usage:
 
-1.  Does `@Nullable` appear directly on that type usage?
-2.  What is the [nullness operator] of that type usage?
-3.  Is it reasonable to assume that `null` will not come "out" of it?
-4.  Is it reasonable to assume that `null` cannot be put "in" to it?
+1. What is the [augmented type] of that type usage?
+2. Do I have to handle the case where `null` comes out of it?
+3. Do I have to prevent `null` from going into it?
+4. Is this type a subtype of that type with respect to nullness?
```

Comment:
I could see an argument that this one doesn't fit as neatly into a list of "Is it nullable?" questions. But I think it's close enough: It's kind of about whether one type is "more nullable" than the other. So I think it works.

Reply:
You could phrase it as "Is this type no more nullable than that type, so that it can be a subtype?"